manifests/windows_service.pp

```diff
@@ -8,11 +8,12 @@
 #
 # [*service_name*]
```

Comment:
Looks like this is using an old style to document parameters. Will that conflict?